ext/RMagick/extconf.rb

```diff
@@ -186,6 +159,32 @@ def exit_failure(msg)
       exit(1)
     end
 
+    def determine_imagemagick_package
+      unless find_executable('pkg-config')
+        exit_failure "Can't install RMagick #{RMAGICK_VERS}. Can't find pkg-config in #{ENV['PATH']}\n"
+      end
+
+      packages = `pkg-config --list-all`.scan(/(ImageMagick\-6[\.A-Z0-9]+) .*/).flatten
+
+      # For ancient version of ImageMagick 6 we need a different regex
+      if packages.empty?
+        packages = `pkg-config --list-all`.scan(/(ImageMagick) .*/).flatten
```

Comment:
Actually, sorry, does this not also cover the case on line 167 above? Can we use this in its place?

Reply:
This one is for just the word `ImageMagick` followed by a space. The other regex is for `ImageMagick-6.Q16` or other versions. And just `ImageMagick` without the space would also match IM 7 and that should not happen yet.

Reply:
Ah, I see. Cool.